### PR TITLE
Add ext. users access to recording to meeting-recording.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9786

Source:
https://support.microsoft.com/office/play-share-and-download-a-meeting-recording-in-teams-7d7e5dc5-9ae4-4b94-8589-27496037e8fa#:~:text=If%20you%20recorded%20or%20organized,share%20it%20however%20you%20like.

### CODE REVIEW

File: Teams/meeting-recording.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ In Microsoft Teams, users can record their Teams meetings and group calls to cap
 When a meeting is recorded, it’s automatically:
 
 - Uploaded to OneDrive or SharePoint
-- Permissioned to the people invited to the meeting
+- Permissioned to the people invited to the meeting (Guests and external attendees can view the recording only if it's explicitly shared with them)
```
Comment (JohanFreelancer9) [inline Teams/meeting-recording.md:34]:
```suggestion
- Permissioned to the people invited to the meeting (guests and external attendees can view the recording only if it's explicitly shared with them)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@dariomws

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@mkbond007 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**